cmd/subcommands/staking.go

```diff
@@ -388,9 +388,14 @@ Create a new validator"
 				}
 			}
 
-			nonce, err := getNonceFromInput(validatorAddress.String(), inputNonce, networkHandler)
-			if err != nil {
-				return err
+			var nonce uint64
```

Comment:
Can probably use a `PreRunE` function, instead of duplicating this many times

Reply:
prerun prob messier